### ISSUE TITLE
Support ignoring modules

### ISSUE THREAD

**Issue by maxfi** [teambit/bit#1243]
Some frameworks like Meteor have internal packages that are provided by the framework directly rather than a package manager. As these can't be resolved it would be great to have the option to simply ignore these dependencies during module resolution. For example, Meteor and Atmosphere (Meteor's package manager) packages all start with `meteor/` so perhaps there could be an option in `bit.json` to ignore certain dependencies:

```json
{
    "resolveModules": {
        "ignore": [
            "meteor/*"
        ]
    },
}
```

_Originally posted by @maxfi in https://github.com/teambit/bit/issues/1019#issuecomment-420160595_

**Comment by GiladShoham** [comment]
@maxfi This is a great feature proposal. We will productize it and prioritize.
We will update the issue.
Thanks!

**Comment by maxfi** [comment]
@GiladShoham any updates on this? Thanks. 🙏

**Comment by GiladShoham** [comment]
@maxfi We have a few big features before we got to this one.
One of them is a new extension system for bit. There is a chance you will even be able to implement this yourself, one the extension system is out.

**Comment by itaymendel** [comment]
@davidfirst , i think this will be resolved as part of overriding deps. right?

**Comment by davidfirst** [comment]
Yes. This is available now with the newly introduced version v14.1.0.
@maxfi , please read more about it here: https://docs.bit.dev/docs/component-dependencies.html#override-dependencies-in-workspace-configuration

**Comment by maxfi** [comment]
For anyone else coming across this, configure your `bit.json` as below:

```json
{
  "env": {},
  "componentsDefaultDirectory": "imports/components/{namespace}/{name}",
  "packageManager": "npm",
  "overrides": {
    "*": {
      "dependencies": {
        "meteor": "-"
      }
    }
  }
}
```
